app/models/concerns/orchestration/dhcp.rb

```diff
@@ -171,17 +171,18 @@ def dhcp_update_required?
       (old&.operatingsystem&.boot_filename(old.host) != operatingsystem&.boot_filename(host)) ||
       ((old.host.pxe_loader == "iPXE Embedded" || host.pxe_loader == "iPXE Embedded") && (old.host.build != host.build)) ||
       (!old.build? && build? && !all_dhcp_records_valid?))
-    # Handle jumpstart
-    # TODO, abstract this way once interfaces are fully used
-    if is_a?(Host::Base) && jumpstart?
-      if !old.build? || (old.medium != medium || old.arch != arch) ||
-          (os && old.os && (old.os.name != os.name || old.os != os))
-        return true
-      end
+
+    if requires_jumpstart_update?
```

Comment:
can we refactor it more like:

```
return true if requires_jumpstart_update?
  false
```

Reply:
Wouldn't even just `requires_jumpstart_update?` do the trick? Or double-negate it to cast it from a truthy/falsey value into a bool?